nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -58,7 +67,7 @@ def target_point(target_type: TargetType,
 
     @staticmethod
     def create_bias_correction_command(node: NNCFNode, bias_value: np.ndarray) -> ONNXBiasCorrectionCommand:
-        create_bias_correction_command(node, bias_value)
+        return create_bias_correction_command(node, bias_value)
```

Comment:
Good catch! My bug.

Reply:
@KodiaqQ Why wasn't it faced during precommit?